.circleci/config.yml

```diff
@@ -52,6 +52,7 @@ jobs:
       - image: circleci/golang:1.13.4-stretch
     steps:
       - checkout
+      - run: cp -r config docker/
```

Comment:
Instead of copying this only in the Circle build, you can modify [the script](https://github.com/palantir/bulldozer/blob/develop/godel/config/dist-plugin.yml#L28-L30) that the build tool, Godel, runs before building the image. That way, it will work for local development too.

Reply:
:heavy_check_mark: